src/pages/Task.tsx

```diff
@@ -158,7 +157,7 @@ const TaskCore: React.FC = () => {
     task?.status?.includes("failed") || task?.status?.includes("timed-out");
 
   // todo: unhide the buildBaronTab when it is production ready
-  const buildBaronIsProductionReady = false;
+  const buildBaronIsProductionReady = true;
```

Comment:
Is this ready or was this accidentally changed?

Reply:
I believe it is ready. There are two outstanding tickets, but I don't think they are blockers.

Reply:
I don't think we should change this flag until we have some tests for this feature, otherwise it will be difficult for us to know when it is not working properly

Reply:
https://jira.mongodb.org/browse/EVG-13094